nucypher/policy/conditions/lingo.py

```diff
@@ -214,6 +217,15 @@ def validate_operator_and_operands(self, data, **kwargs):
                 conditions=operands, field_name="operands"
             )
 
+            if operator == CompoundCondition.AT_LEAST_OPERATOR:
+                number_of_operands = len(operands)
+                threshold = data.get("threshold", 0)
+                if threshold < 1 or threshold > number_of_operands:
+                    raise ValidationError(
+                        field_name="threshold",
+                        message=f"Threshold must be between 1 and number of operands ({number_of_operands})",
+                    )
```

Comment:
Do we need to also ensure:
1.  "threshold" is always set for AT_LEAST (effectively it is required for this operator)
2. "threshold" is never set, i.e. it should be `None`, for any other compound condition operator 

So something like:

```suggestion
            threshold = data.get("threshold")
            if operator == CompoundCondition.AT_LEAST_OPERATOR:
                if threshold is None:
                    raise ValidationError(
                        field_name="threshold",
                        message=f"Threshold must be specified for {operator} operator",
                    )
                number_of_operands = len(operands)
                if 1 <= threshold <= number_of_operands:
                    raise ValidationError(
                        field_name="threshold",
                        message=f"Threshold must be between 1 and number of operands ({number_of_operands})",
                    )
            elif threshold is not None:
                raise ValidationError(
                    field_name="threshold",
                    message=f"Threshold is not applicable for {operator} operator"
                )
```

Reply:
✅